.changeset/eleven-meals-bow.md

```diff
@@ -0,0 +1,35 @@
+---
+'@openfn/language-googlehealthcare': major
```

Comment:
hmm. I think this is a major for common (because `expandReferences) has changed, but I think it's a patch for everyone else (because they don't export expandReferences)

So you probably need two changesets with slightly different notes?